charts/simple-app/values.yaml

```diff
@@ -177,6 +177,15 @@ envFrom: []
   # - configMapRef:
   #     name: my-config-vars
 
+# -- (`Map`) Map of environment variables to plaintext secrets or KMS encrypted secrets.
+secrets: {}
+
+# -- (String) AWS region where the KMS key is located
+kmsSecretsRegion:
+
+# -- (String) Secrets Engine determines the type of Secret Resource that will be created (`KMSSecret`, `Secret`). kms || plaintext are possible values.
+secretsEngine: plaintext
```

Comment:
Open to switching this to being a bool flag. Maybe `kmsEncypted` flag ?

Reply:
I think right now this is good.. there is another secret engine we use too sometimes, so we may choose to add that in here at some point.